GitVersionCore/VersionCalculation/BaseVersionCalculator.cs

```diff
@@ -0,0 +1,31 @@
+﻿namespace GitVersion.VersionCalculation
```

Comment:
Can we create a .DotSettings files which does that.

It is a pain fighting tooling